src/wallet/cc_wallet/cc_utils.py

```diff
@@ -164,13 +164,13 @@ def spend_bundle_for_spendable_ccs(
 
     # now add solutions to consume the lock coins
 
-    for _ in range(N):
-        prev_index = (_ - 1) % N
-        prev_coin = spendable_cc_list[prev_index].coin
-        this_coin = spendable_cc_list[_].coin
-        subtotal = subtotals[_]
-        coin_solution = coin_solution_for_lock_coin(prev_coin, subtotal, this_coin)
-        coin_solutions.append(coin_solution)
+    # for _ in range(N):
```

Comment:
What's happening here? A comment explaining would be nice.

Reply:
I'll just delete outright, you no longer need to create a spend for the lock because the lock is an announcement